consensus/ledger_validator/ledger_validator.c

```diff
@@ -256,8 +259,10 @@ retcode_t iota_consensus_ledger_validator_update_snapshot(ledger_validator_t con
           goto done;
         }
       }
-      if ((ret = iota_snapshot_apply_patch(lv->milestone_tracker->latest_snapshot, &delta, milestone->index)) !=
-          RC_OK) {
+
+      if ((ret = iota_milestone_service_replay_milestones(tangle, &lv->milestone_service,
+                                                          &lv->milestone_tracker->snapshots_provider->latest_snapshot,
+                                                          milestone->index)) != RC_OK) {
```

Comment:
Replaying includes applying and some other stuff, and in that _lv_ case, it will also replay one milestone only, the reason is the "other stuff" of course, which includes for now only updating entry_solid points but will include `seen_milestones` too